packs/feats/starlit-sentinel-dedication.json

```diff
@@ -25,6 +25,61 @@
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
         "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.StarlitSentinel.Constellation.Archer",
+                        "value": "archer"
+                    },
```

Comment:
Instead of a simple choice do an object
```json
"value": {
    "constellation": "archer",
    "damageType": "piercing"
}
```
Then you can store those in actor flags and reference that instead of having a million DamageAlterations. If we need a roll option for the constellation we can set one with the RollOption RE.